service/src/main/kotlin/fi/espoo/evaka/varda/VardaUpdateServiceV2.kt

```diff
@@ -624,7 +645,8 @@ WITH child_fees AS (
     fdc.child_id,
     fd.valid_during
   FROM fee_decision fd JOIN fee_decision_child fdc ON fd.id = fdc.fee_decision_id 
-  ${if (startingFrom != null) " WHERE fd.sent_at >= :startingFrom" else ""}  
+  ${if (startingFrom != null) " WHERE fd.sent_at >= :startingFrom" else ""}
+  ${if (feeDecisionStatus != null) " WHERE fd.status >= :feeDecisionStatus" else ""}  
```

Comment:
```suggestion
  ${if (feeDecisionStatus != null) " WHERE fd.status = :feeDecisionStatus" else ""}  
```

Reply:
Maybe we should also add a check that not both params `feeDecisionStatus` and `voucherValueDecisionStatus` are null? Eg something like `check(feeDecisionStatus != null || voucherValueDecisionStatus != null)`